docs/NIC.md

```diff
@@ -10,24 +10,29 @@ Most commonly, NIC is used interactively like so:
 NIC 2.0 - New Instance Creator
 ------------------------------
   [1.] iphone/activator_event
-  [2.] iphone/application_modern
-  [3.] iphone/cydget
-  [4.] iphone/flipswitch_switch
-  [5.] iphone/framework
-  [6.] iphone/ios7_notification_center_widget
-  [7.] iphone/library
-  [8.] iphone/notification_center_widget
-  [9.] iphone/preference_bundle_modern
-  [10.] iphone/theme
-  [11.] iphone/tool
-  [12.] iphone/tweak
-  [13.] iphone/xpc_service
-Choose a Template (required): 12
+  [2.] iphone/activator_listener
+  [3.] iphone/application_modern
+  [4.] iphone/application_swift
+  [5.] iphone/control_center_module-11up
+  [6.] iphone/cydget
+  [7.] iphone/flipswitch_switch
+  [8.] iphone/framework
+  [9.] iphone/library
+  [10.] iphone/notification_center_widget
+  [11.] iphone/notification_center_widget-7up
+  [12.] iphone/preference_bundle_modern
+  [13.] iphone/theme
+  [14.] iphone/tool
+  [15.] iphone/tool_swift
+  [16.] iphone/tweak
+  [17.] iphone/tweak_with_simple_preferences
+  [18.] iphone/xpc_service
+Choose a Template (required): 16
 Project Name (required): Example
 Package Name [com.yourcompany.example]: tech.theos.example
 Authour/Maintainer Name [Craig Federighi]: Craig Federighi <notfederighi@theos.tech>
```

Comment:
Let’s change these to theos.dev - I think we had theos.tech for a while but it wasn’t really used for anything, we since switched to theos.dev which will remain our permanent domain.